### DISCUSSION TITLE
add postgres driver through helm chart

### DISCUSSION THREAD

**Discussion by adeel-ansari** [apache/doris#43472] [A - General / Q&A] [answered]
Hi,

I'm trying to use the SystemInitialization section of the doris helm chart to download the postgres driver.  However, I'm running into errors.  I'm trying something simple to troubleshoot:

`        systemInitialization:
          initImage: "selectdb/alpine:latest"
          command: ["/bin/sh", "-c", "echo hello"]`

Even that is giving the following error:

> Error: failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/bin/sh -c echo hello": stat /bin/sh -c echo hello: no such file or directory: unknown

Do you know where I may be going wrong?  The full set of commands I need to perform are the following but I'm stuck at the moment:

        #       cd /opt/apache-doris && \
        #       mkdir -p be/jdbc_drivers && \
        #       curl -o be/jdbc_drivers/postgresql-42.5.6.jar https://jdbc.postgresql.org/download/postgresql-42.5.6.jar

**Comment by intelligentfu8** [comment]
the InitContainer for initialing environment of doris start. "/bin/sh" is not needed. although, download successes, the init Container is not shared with Doris container, so please share why you want to download the PostgreSQL jar.

**Reply by adeel-ansari** [reply]
Hi, this was to query a postgres database through Doris.  I was trying to download the jar file using the Helm chart instead of entering the pod but wasn't able to.  I think we need the jar file as mentioned here: https://doris.apache.org/docs/3.0/lakehouse/database/postgresql

**Comment by intelligentfu8** [comment, marked as answer]
Sorry, The drivers about the catalog do not support downloading at the start time. we will support it soon. now please build the image by yourself.

**Reply by adeel-ansari** [reply]
Thank you for confirming.  I'll try your suggestion.
